test/test_op.py

```diff
@@ -58,26 +58,26 @@ def plus_one(x):
     assert hasattr(unary, 'plus_one')
     assert unary.plus_one.types == {'INT8', 'INT16', 'INT32', 'INT64',
                                     'UINT8', 'UINT16', 'UINT32', 'UINT64',
-                                    'FP32', 'FP64'}
+                                    'FP32', 'FP64', 'BOOL'}
     v = Vector.new_from_values([0, 1, 3], [1, 2, -4], dtype=dtypes.INT32)
     v << v.apply(unary.plus_one)
     result = Vector.new_from_values([0, 1, 3], [2, 3, -3], dtype=dtypes.INT32)
     assert v.isequal(result)
 
 
 def test_unaryop_udf_bool_result():
-    pytest.xfail('not sure why numba has trouble compiling this')
```

Comment:
Good news!  The hack to change BOOL to INT8 during computation appeared to work.  Well, at least this test passes now.